pkg/bypass4netns/bypass4netns.go

```diff
@@ -1018,10 +1014,13 @@ func (h *notifHandler) startBackgroundMultinodeTask(ready chan bool) {
 						hostAddr := fmt.Sprintf("%s:%d", h.multinode.HostAddress, v.HostPort)
 						// Remove entries with timeout
 						// TODO: Remove related entries when exiting.
-						opts := &client.SetOptions{
-							TTL: time.Second * 15,
+						lease, err := h.multinode.etcdClient.Grant(gocontext.TODO(), 15)
```

Comment:
Replacing `gocontext.TODO()` in `Grant`,`Put`, and `Get` with context created with `context.WithCancel()` is a better way to handle timeout.